k8s.tf

```diff
@@ -12,18 +12,6 @@ locals {
   }
 }
 
-resource "kubernetes_config_map" "beekeeper" {
```

Comment:
Thinking about this removal a bit more, I think switching from a config map to a direct env variable makes sense given that I don't think that we have Beekeeper setup to auto reload & populate it's properties & config unless the service restarts.

It seems that there's really no benefit to that since we're forced to do a full restart anyways, having the config map becomes more of a preferential choice than anything.

Reply:
Using the env variable directly matches what we already have in the helm chart. It would be nice to use this though, i think the spring boot apps would work if they are reading all the vars directly from the environment. But I think its easier for now to match what we already have to get this through. (Without this change the terraform plan was throwing a few errors).